src/Directives/DefaultDirective.php

```diff
@@ -0,0 +1,148 @@
+                public function name(): string
+                {
+                    return Directive::GEOLOCATION;
+                }
+
+            },
+            Directive::GYROSCOPE => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::GYROSCOPE;
+                }
+            },
+            Directive::MAGNETOMETER => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::MAGNETOMETER;
+                }
+            },
+            Directive::MICROPHONE => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::MICROPHONE;
+                }
+            },
+            Directive::MIDI => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::MIDI;
+                }
+            },
+            Directive::PAYMENT => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::PAYMENT;
+                }
+            },
+            Directive::PICTURE_IN_PICTURE => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::PICTURE_IN_PICTURE;
+                }
+            },
+            Directive::SPEAKER => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::SPEAKER;
+                }
+            },
+            Directive::SYNC_XHR => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::SYNC_XHR;
+                }
+            },
+            Directive::USB => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::USB;
+                }
+            },
+            Directive::WAKE_LOCK => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return Directive::WAKE_LOCK;
+                }
+            },
+            Directive::XR, Directive::VR => new class extends Directive implements DirectiveContract {
+                public function name(): string
+                {
+                    return 'vr';
+                }
+            },
+            default => throw InvalidDirective::notSupported($directive),
```

Comment:
When using a directive which isn't defined here, throw an exception. Same as the prior `Directive::isValid()` method.